util/webhook/webhook_test.go

```diff
@@ -404,6 +404,498 @@ func TestGitHubCommitEvent_Hydrate(t *testing.T) {
 	hook.Reset()
 }
 
+func TestGitHubCommitEvent_SyncSourceRefresh(t *testing.T) {
```

Comment:
The unit tests should validate that `SetNewRevisionManifests` has been called when the source matches, but the path do not match any changed files